doc/source/user/workbench.rst

```diff
@@ -0,0 +1,178 @@
+.. _workbench:
+
+Setup WATTS in NEAMS Workbench Environment
+------------------------------------------
+
+The `NEAMS Workbench <https://www.ornl.gov/onramp/neams-workbench>` is a tool
+that facilitates the use of multiple tools in analysis
+by providing a commom user interface for model creation, review, execution, output,
+and visualization for integrated codes. Instructions on how to download and install
+Workbench can be found on the Workbench
+`Gitlab repository <https://code.ornl.gov/neams-workbench/downloads>`.
+
+To set up WATTS in the Workbench environment, user needs to first provide the path to
+where Workbench is installed in `workbench.sh` under the `scripts` directory. User can
+then run `setup_conda_wb.sh` under the same directory to set up WATTS within
+the Workbench environment.
+
+Optional: To install OpenMC in the Workbench environemnt, run `setup_openmc.sh`.
+
+To finish setting up WATTS, open Workbench and go to the `File` tab at the top-left corner
+then select `Configurations` from the drop menu. In the `Application configurations`
+window, click `Add` on the top most row then select `WATTS` from the pop-up window.
+In the list of `Application Options`, input the path of `watts_ui.py` to `Executable`.
+The file should exist by default in `/watts/src/watt_ui/`. Next, click `Load Grammar`.
+Click `OK` to complete the setup.
+
+Optional: Environment variables can be added be added under `Run Environment`.
+
+WATTS Workbench Input File
+++++++++++++++++++++++++++
+
+To create a new WATTS input file in Workbench, go to `File`, select `New File`,
+and select `WATTS files (*.son)` from the drop down menu. An existing `*.son`
+file can also be dragged and dropped to the Workbench window.
+
+The WATTS input file utilizes a hierarchical block-based system where a block
+is defined with opening and closing curly braces `{ }`. A `watts` block is first
+created within which other blocks and variables can be defined. Within the `watts`
+block, `workflow_dir` needs to be defined as the path to the working directory
```

Comment:
I don't understand why this directory would need to be defined.

Reply:
Removed `workflow_dir`